src/api/app.py

```diff
@@ -95,13 +98,20 @@ async def proxy(request: Request, path: str):
         headers["Authorization"] = f"Bearer {access_token}"
 
         try:
+            content = await request.body()
+            content = json.loads(content)
+
+            if USE_MODEL_MAPPING:
+                request_model = content.get("model", None)
+                content["model"] = get_model(PROVIDER, REGION, request_model)
+                content = json.dumps(content)
```

Comment:
Let's rename one of these `content` variables
```suggestion
            content = await request.body()
            data = json.loads(content)

            if USE_MODEL_MAPPING:
                request_model = data.get("model", None)
                data["model"] = get_model(PROVIDER, REGION, request_model)
                content = json.dumps(data)
```